src/appleseed/renderer/modeling/bsdf/bsdf.h

```diff
@@ -193,11 +193,11 @@ class APPLESEED_DLLSYMBOL BSDF
         SamplingContext&            sampling_context,
         const void*                 data) const;
 
-    // Modulate transmittance by medium absorption over the given distance.
+    // Modulate value by medium absorption over the given distance.
```

Comment:
Out of curiosity, why the change? Can we modulate anything else than a transmittance?